src/gfx/math.hpp

```diff
@@ -40,6 +40,11 @@ template <size_t Alignment> inline constexpr size_t alignTo(size_t size) {
   }
 }
 
+template <typename T> inline bool isWithinRange(T val, T target, T tolerance = 0.05f) {
```

Comment:
Isn't that a big default? Usually I have seen epsilon be around 1E-3 or lower.

Reply:
It's okay for the current usage, renamed it to justify the larger default epsilon